docs/knowledgebase/getting-started/index.md

```diff
@@ -43,6 +43,15 @@ export OPENSSL_LIB_DIR="/usr/lib/openssl-1.0"
 export OPENSSL_INCLUDE_DIR="/usr/include/openssl-1.0"
 ```
 
+### Fedora
+
+Run these commands from a terminal:
+
+```bash
+sudo dnf update
+sudo dnf install automake clang clang-devel cmake curl gcc gcc-c++ git kernel-devel make openssl-devel pkgconf-pkg-config zlib-devel
```

Comment:
```suggestion
sudo dnf install clang clang-devel curl gcc gcc-c++ git kernel-devel openssl-devel pkgconf-pkg-config zlib-devel
```
Not sure if you really need all of this either TBH... `curl` could be optional for instance. clang and gcc/g++ are both needed? cc @jimmychu0807 who should we ask about base required deps for us beyond what rust needs?